win_configure/msi/wix/create_msi.bat

```diff
@@ -0,0 +1,35 @@
+@echo off
+REM This script will generate msi installer using Wix toolset
+
+REM Defining needed variables 
+
+SET PBS_prefix=''
+@setlocal enableextensions enabledelayedexpansion
+set variable=%~dp0
+if "x!variable:~-29!"=="xpbspro\win_configure\msi\wix\" (
+	set variable=!variable:~0,-29!
+) else (
+@echo "Failed to parse PBS prefix location"
+goto theend
+)
+
+SET PBS_prefix=!variable!
+
+SET TOPDIR=%PBS_prefix%PBS
+
+REM Calling wix compiler
+
+candle Product.wxs temp\pbsproexec.wxs temp\pbsprohome.wxs -ext WixUtilExtension.dll -ext WixUIExtension -d"pbsproexec=%TOPDIR%\exec" -d"pbsprohome=%TOPDIR%\home"
+@if NOT %ERRORLEVEL% == 0 goto theend
+
+REM Caling Wix linker
+light -ext WixUIExtension -ext WiXUtilExtension pbsproexec.wixobj Product.wixobj pbsprohome.wixobj -o PBSPro.msi -d"pbsproexec=%TOPDIR%\exec" -d"pbsprohome=%TOPDIR%\home"
```

Comment:
As such no manufacturer is building and distribution the msi installer, Version number is not needed. 
+
Even i didn't have any specific requirement to add.
+
If the builder decides to add version, its their choice to choose. 

Here, Its a solution to build package, not for distribution.